Makefile

```diff
@@ -1,12 +1,34 @@
-# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright 2019-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-BUILDDIR ?= $(abspath build)
-OUTDIR   ?= $(abspath blobs)
+BUILDDIR 	      ?= $(abspath build)
+OUTDIR		      ?= $(abspath blobs)
+ARCH_x86_64 	       = x86_64
+ARCH_aarch64	       = arm64
+KERN_IMAGE_x86_64      = bzImage
+KERN_IMAGE_aarch64     = Image
+CONFIG_x86_64	       = microvm-kernel-config-x86_64
+CONFIG_aarch64	       = microvm-kernel-config-aarch64
+DEFAULT_CROSS_COMPILER = /usr/bin/aarch64-linux-gnu-
+
+ifndef ARCH
+	ARCH=$(ARCH_x86_64)
```

Comment:
I think using the local arch would be recommended instead of using x86_64 by default.